### ISSUE TITLE
@babel/plugin-transform-runtime not applying to imported modules

### ISSUE THREAD

**Issue by bendemboski** [embroider-build/embroider#1351]
1. Create a new application (I used `ember-cli@4.10.0`) and set it up to build with embroider
2. `ember serve`
3. Navigate to localhost:4200

In the devtools open `/app.js` and observe that it imports `/node_modules/@babel/runtime/helpers/esm/defineProperty.js`, then open `/node_modules/@glimmer/component/-private/component.js` and observe that it defines a `_defineProperty` function (and others) inline.

Similarly, if you make the app import from `tracked-built-ins`, its files will include inline `_defineProperty`s, so it looks like no imported modules have `@babel/plugin-transform-runtime` applied, while modules in the app itself do.

Was #614 meant to apply to imported modules as well as app modules? Or is there a reason we're not applying it to imported modules?

**Comment by ef4** [comment]
The first thing to check here is whether the inline helpers are getting emitted in the first stage (when v1 addons get rewritten to v2) or in the last stage (when the whole app gets built by webpack). You can check by seeing if they're already in there in the copies of the addons present in the tempdir that gets printed as "Building into... " as the build starts.

**Comment by bendemboski** [comment]
@ef4 yes, the inline helpers are present in the `<that temp dir>/node_modules/@glimmer/component/-private/component.js`
